libs/product/src/drivers/magento/transforms/product-transformers.spec.ts

```diff
@@ -35,6 +36,18 @@ describe('DaffMagentoProductTransformers', () => {
 				expect(transformMagentoProduct(magentoBundledProduct, mediaUrl).type).toEqual(DaffProductTypeEnum.Composite);
 			});
 		});
+
+		describe('when the product is a configurable product', () => {
+			let magentoConfigurableProduct: MagentoConfigurableProduct;
+
+			beforeEach(() => {
+				magentoConfigurableProduct = new MagentoConfigurableProductFactory().create();
+			});
+
+			it('should return a composite product', () => {
```

Comment:
Is composite the same thing as configurable?

Reply:
no, that's a mistake. Fixed